src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs

```diff
@@ -29,6 +29,31 @@ public class Perf_File
         private Dictionary<int, string> _filesToRead;
         private string[] _linesToAppend;
         private Dictionary<int, string> _textToAppend;
+        private string _moveSource;
+        private string _moveDestination;
+
+        [IterationSetup(Target = nameof(Move))]
```

Comment:
we try not to use IterationSetup if possible: https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#iterationsetup

In case of move benchmark, we could just swap the files:

```cs
private ulong counter;

[Benchmark]
public void Move()
{
     if (counter++ % 2 == 0)
         File.Move(_moveSource, _moveDestination);
     else
         File.Move(_moveDestination, _moveSource);
}
```